vcluster/configure/vcluster-yaml/control-plane/other/advanced/workload-service-account.mdx

```diff
@@ -10,12 +10,30 @@ import TenancySupport from '../../../../../_fragments/tenancy-support.mdx';
 
 <TenancySupport hostNodes="true" privateNodes="true" />
 
-This has similar usage to `controlPlane.advanced.serviceAccount` but for use with workloads deployed to the virtual cluster.
+The workloadServiceAccount allows you to enforce the use of a specific ServiceAccount for all vCluster workload pods running in the host cluster.
+For example, you can attach a shared imagePullSecret to the ServiceAccount so that all synced pods use the same secret when pulling container images.
+```yaml
+controlPlane:
+  advanced:
+    workloadServiceAccount:
+      enabled: true
+      name: image-pull-sa
+      imagePullSecrets:
+        - name: workload-service-account-image-pull-secret
+```
+:::note
+The pod specification (spec) of the synced workloads should not define its own imagePullSecrets.
+This is because imagePullSecrets are only copied from the ServiceAccount to the pod when the pod specification does not already include them.
 
-:::info
-If [syncing ServiceAccount resources from the virtual cluster to the host cluster](../../../sync/to-host/advanced/service-accounts.mdx) is enabled, the `workloadServiceAccount` setting is ignored.
+The referenced imagePullSecrets must also exist in the host namespace where the vCluster’s workload pods are synced.
+
+An easy way to achieve this is by using Virtual Cluster Templates.
```

Comment:
Maybe adding a link to the Virtual Cluster Templates documentation would help users understand how to create the imagePullSecret in the host namespace
```suggestion
An easy way to achieve this is by using [Virtual Cluster Templates](../../../../../../platform/administer/templates/create-templates.mdx).
```